web/src/main/scala/net/wiringbits/forms/ResendVerifyEmailFormData.scala

```diff
@@ -0,0 +1,40 @@
+package net.wiringbits.forms
+
+import net.wiringbits.api.models.SendEmailVerificationToken
+import net.wiringbits.common.models.{Captcha, Email}
+import net.wiringbits.webapp.utils.slinkyUtils.forms.{FormData, FormField}
+
+case class ResendVerifyEmailFormData(
```

Comment:
This component needs a test. You can see that we create a test for each FormData, it help us to know the behavior of the form